.github/workflows/test-pr-external-dns.yaml

```diff
@@ -0,0 +1,39 @@
+name: Unit Test - External DNS
+
+on:
+  pull_request:
+    types:
+      - ready_for_review
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - "addons/packages/external-dns/**/*"
```

Comment:
I would also add modification of this file/action itself in the path list

Reply:
That makes sense. I've added it to the paths.